packages/shared/src/styles/custom.ts

```diff
@@ -0,0 +1,9 @@
+// this file was created to contain all custom temporary colors (outside the guideline)
+// and it is ideal to have this file to contain fewer colors as possible
+// we should eventually convert the custom colors to be part of the guideline once it is deemed to be used frequently
+
+export const feedSurveyBg =
+  'linear-gradient(180deg, rgba(255, 233, 35, 0.08) 0%, rgba(252, 83, 141, 0.08) 50%, rgba(113, 71, 237, 0.08) 100%)';
+
+export const feedSurveyBorder =
+  'linear-gradient(180deg, rgba(255, 233, 35, 1) 0%, rgba(252, 83, 141, 1) 50%, rgba(113, 71, 237, 1) 100%)';
```

Comment:
```suggestion
  'linear-gradient(180deg, var(--theme-accent-cheese-default) 0%, var(--theme-accent-bacon-default) 50%, var(--theme-accent-onion-default) 100%)';
```

Reply:
We can use the variables instead of the rgb values :)

Reply:
Nice. Yeah that is definitely much better. Thank you.